.github/workflows/ci.yml

```diff
@@ -16,6 +16,9 @@ jobs:
   build-x86_64-darwin:
     uses: ./.github/workflows/build-x86_64-darwin.yml
 
+  build-aarch64-linux:
+    uses: ./.github/workflows/build-aarch64-darwin.yml
```

Comment:
?

(`aarch64-linux` in the job name, and `aarch64-darwin` in the filename)

Reply:
I figured it was something like this